django_netjsongraph/templates/netjsongraph/base.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8">
     {% block style %}
         <link href="{% static 'netjsongraph/src/netjsongraph.css' %}" rel="stylesheet">
-        <link href="{% static 'netjsongraph/style.css' %}" rel="stylesheet">
+        <link href="{{ custom_css }}" rel="stylesheet">
```

Comment:
Try this:

`{% static VISUALIZER_CSS %}` or `{% static '{{ VISUALIZER_CSS }}' %}`